sklearn/metrics/_plot/roc_curve.py

```diff
@@ -118,11 +142,25 @@ def plot(self, ax=None, *, name=None, **kwargs):
 
         line_kwargs.update(**kwargs)
 
+        chance_level_line_kwargs = {
+            "label": "Chance level (AUC = 0.5)",
+            "color": "k",
+            "linestyle": "--",
+        }
+
+        if chance_level_kwargs is not None:
+            chance_level_line_kwargs.update(**chance_level_kwargs)
+
         import matplotlib.pyplot as plt
 
         if ax is None:
             fig, ax = plt.subplots()
 
+        if plot_chance_level:
+            (self.chance_level_,) = ax.plot((0, 1), (0, 1), **chance_level_line_kwargs)
+        else:
+            self.chance_level_ = None
```

Comment:
Let's delay this plotting after having plot the other ROC curve. Otherwise the chance level appear in the legend as the first item. I would expect it to always be the last one.